npm/vite-plugin-cypress-esm/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@cypress/vite-plugin-cypress-esm",
+  "version": "0.0.0-development",
+  "description": "Make ESM Modules mutable in the browser with Cypress and Vite",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc || echo 'built, with type errors'",
+    "build-prod": "tsc || echo 'built, with type errors'",
+    "check-ts": "tsc --noEmit",
+    "cypress:open": "node ../../scripts/cypress open --project . --component",
+    "test": "node ../../scripts/cypress run --project . --browser chrome --component",
+    "watch": "tsc -w",
+    "lint": "eslint --ext .js,.ts,.json, ."
+  },
+  "dependencies": {
+    "debug": "^4.3.4",
+    "semver": "^7.3.2"
```

Comment:
Is `semver` actually used anywhere? I don't see it

Reply:
I don't think so, I think it can be removed. I'll  remove it.